client/js/shout.js

```diff
@@ -382,6 +382,28 @@ $(function() {
 		}
 	}
 
+	if ($.cookie("timestamp")) {
+		$("#timestamp").val($.cookie("timestamp")).change();
+	} else {
+		$("#timestamp").val("HH:mm").change();
+	}
+
+	$("#timestamp").on("change", function() {
+		var timestamp = $(this).val();
+		Handlebars.unregisterHelper("tz");
```

Comment:
Is there really a need to re-register the handler? Can't you just make `timestamp` variable global?